docs/usage.md

```diff
@@ -88,12 +94,20 @@ with `params.yaml` containing:
 ```yaml
 input: './samplesheet.csv'
 outdir: './results/'
-genome: 'GRCh37'
+genome: 'GRCh38'
 <...>
 ```
 
 You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-co.re/launch).
 
+To change the taxon or taxonomic subtree which is classified by kraken2 as contamination use the `tax2filter` parameter (default **_Homo_**). The taxon has to be in the kraken2 database used, which can be specified using the `kraken2db` parameter.
+
+To change the organism(s) which should be validated as contamination(s) by blasting against a database, you have to provide a fasta from which the blastn database is built using the `fasta` parameter. Also, if just one reference genome is needed for blastn and it is in [igenomes.config](../conf/igenomes.config) use the according name (e.g. `'GRCh38'`) as `genome` parameter.
+
+Skipping blastn can be done by using `--skip_blastn`.
+
+Optionally enabling the filter can be done by using `--enable_filter`. By default the raw reads are taken for filtering. If you want to use the preprocessed reads use the `--filter_trimmed` flag. By default the blastn output is taken for filtering. To use the kraken2 output for filtering while still using blast for validation use the `--filter_with_kraken2` flag. If you enable the filter and use the `--skip_blastn` option, kraken2 output is taken automatically for filtering.
```

Comment:
this sentence is confusing to me because you write once that raw reads are taken for filtering by default and then blastn outputs. If I understand this correctly, these are two different filtering steps, so it should be stated as such. 

In general, try not to define too many parameters here, we have the parameters page for that and it's easier to keep in sync, if you have everything on one page.